akka-actor/src/main/scala/akka/actor/Scheduler.scala

```diff
@@ -74,6 +74,9 @@ trait Scheduler {
    * of the function executions). In such cases, the actual execution interval
    * will differ from the interval passed to this method.
    *
+   * If the function throws an exception the repeated scheduling is aborted,
```

Comment:
It is logged. In the test I added:

    [ERROR] [09/16/2015 11:40:12.744] [LightArrayRevolverSchedulerSpec-akka.actor.default-dispatcher-3] [TaskInvocation] TEST (akka.actor.SchedulerSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$apply$mcV$sp$5$$anon$1)

I think it is logged via the `ExecutionContext.reportFailure` and for our dispatchers we publish that Error to the eventStream